docs/execution_providers/OpenVINO-ExecutionProvider.md

```diff
@@ -140,27 +140,30 @@ Below topologies from ONNX open model zoo are fully supported on OpenVINO Execut
 
 ## CSharp API
 
-*To use csharp api for openvino execution provider create a custom nuget package.
+To use csharp api for openvino execution provider create a custom nuget package.
 
-#Windows
+1.Windows
 
-Build windows build with --build_csharp option. 
+Build a custom nuget package for windows.
+```
 .\build.bat --config Debug --build --use_openvino $Device --build_csharp
-
-Build Custom Nuget Package
 msbuild csharp\OnnxRuntime.CSharp.proj /p:Configuration=Debug /t:CreatePackage
 
-#Linux
-We do not have a current build process to build directly in Linux. But we can
+```
+
+2.Linux
```

Comment:
leave a space after '1.' to make it appear as a heading (https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#lists)

Reply:
done